assets/src/application/ui/display/DebugInfo.tsx

```diff
@@ -1,15 +1,14 @@
-import React, { CSSProperties } from 'react';
-import { Collapse } from 'antd';
-
-const { Panel } = Collapse;
-
-const dataStyle: CSSProperties = {
-	borderRadius: '5px',
-	boxSizing: 'border-box',
-	padding: '1em 2em',
-	color: '#a9ce47',
-	backgroundColor: '#26203d',
-};
+import React from 'react';
+import { Button, Collapse } from '@chakra-ui/core';
+import styled from '@emotion/styled';
+
+const Pre = styled.pre`
+	border-radius: 5px;
+	box-sizing: border-box;
+	padding: 1em 2em;
+	color: #a9ce47;
+	background-color: #26203d;
+`;
```

Comment:
these styles were applied taking into account that this component is rendered only in dev env